ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6623,69 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+SEDPM:
+  description: "Supply, exhaust pressure monitoring"
+  is_abstract: true
+  uses:
+  - supply_air_differential_pressure_sensor
+  - exhaust_air_differential_pressure_sensor
+  implements:
+  - MONITORING
+
+DXC:
+  description: "Compressor with 3 stage cooling control, single status"
+  is_abstract: true
+  uses:
+  - cooling_percentage_command
+  - compressor_run_command_1
+  - compressor_run_command_2
+  - compressor_run_command_3
+  - compressor_run_status
+  - failed_compressor_alarm
+  implements:
+  - CONTROL
+
+SEMC:
+  description: "Season mode control"
+  is_abstract: true
+  uses:
+  - supply_air_temperature_setpoint_1
+  - supply_air_temperature_setpoint_2
+  - winter_outside_temperature_setpoint
+  - summer_outside_temperature_setpoint
+  - supply_air_deadband_temperature_setpoint
+  - season_mode
+  implements:
+  - CONTROL
+
+HWVPM:
```

Comment:
If it has a command then it should have a sensor. See if you can use `HWVM` and flag your unit as missing the sensor if you don't see it.